### PR TITLE
reduce model size for dplr unittest

### PR DESCRIPTION
**Pull request by Yi-FanLi**
This PR uses small models to test dplr.

### CODE REVIEW

File: source/lmp/tests/lrmodel.pbtxt
Comment (njzjz) [inline source/lmp/tests/lrmodel.pbtxt:1]: Why does the model have more lines when you reduce the size?
image

Comment (njzjz) [reply source/lmp/tests/lrmodel.pbtxt:1]: > while the previous one is 20 M

??

Comment (njzjz) [reply source/lmp/tests/lrmodel.pbtxt:1]: Can you reduce 1.3MB to <1.0MB?

Comment (Yi-FanLi) [reply source/lmp/tests/lrmodel.pbtxt:1]: Yes, now it is reduced to 950 K.

**Review by njzjz — APPROVED** [review]

**Pull request merged**